clone-extension.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+EXTENSION=$1
+BRANCH=$2
+
+git clone https://github.com/wikimedia/mediawiki-extensions-${EXTENSION}.git -b ${BRANCH} ${EXTENSION}
```

Comment:
If we run into problems (speed or resource limit), we can still clone from the official repo

https://gerrit.wikimedia.org/r/mediawiki/extensions/$EXTENSiON

Reply:
Yes we could do: If clone from git doesn't work -> fallback on gerrit. But I think it's super unlikely. The worst that could happen is that we are stuck with yesterdays image, that wouldn't be so bad.

Reply:
I mean gerrit instead of github

Reply:
Now cloning from Gerrit by default, fallback on Github.